jgscm/__init__.py

```diff
@@ -209,6 +209,10 @@ class GoogleStorageContentManager(ContentsManager):
             """
                          )
 
+    def __init__(self, *args, **kwargs):
+        self._client = None
```

Comment:
Would be nice to have a comment about what is this client

Reply:
Done